addons/mail/static/src/components/messaging_menu/messaging_menu.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.MessagingMenu" owl="1">
         <div class="o_MessagingMenu dropdown" t-att-class="{ 'show': messagingMenu ? messagingMenu.isOpen : false, 'o-mobile': messaging ? messaging.device.isMobile : false }" t-attf-class="{{ className }}" t-ref="root">
             <a class="o_MessagingMenu_toggler dropdown-toggle o-no-caret o-dropdown--narrow" t-att-class="{ 'o-no-notification': messagingMenu ? !messagingMenu.counter : false }" href="#" title="Conversations" role="button" t-att-aria-expanded="messagingMenu and messagingMenu.isOpen ? 'true' : 'false'" aria-haspopup="true" t-on-click="_onClickToggler">
-                <i class="o_MessagingMenu_icon oi oi-large oi-discuss" role="img" aria-label="Messages"/>
+                <i class="o_MessagingMenu_icon fa oi-large fa-comments" role="img" aria-label="Messages"/>
```

Comment:
Is it intended to keep `oi-large` with `fa` icons?

Reply:
Technically  `oi-large` and `fa-lg` provide the same visual result now.
They have been optimised to get a `16px` icon starting from our `13px` default font-size (before  `fa-lg` returned a `17.33px` icon).

Anyways, `fa-lg` should be used in this case, I agree.

As said, nothing changes visually, but makes sense to use  `fa-lg` when in conjunction with `fa` icons.
Thanks